src/main/java/cat/udl/eps/softarch/demo/domain/Shelter.java

```diff
@@ -1,11 +1,10 @@
 package cat.udl.eps.softarch.demo.domain;
 
-import jakarta.persistence.Column;
-import jakarta.persistence.Entity;
+import com.fasterxml.jackson.annotation.JsonIdentityReference;
+import jakarta.persistence.*;

```

Comment:
It would be best if you imported the minimum dependencies that you need in your project. 
💡 TIP: Use the shortcut ⌃⌥O (macOS) / Ctrl+Alt+O.

Reply:
Done